pkgs/development/tools/build-managers/meson/setup-hook.sh

```diff
@@ -20,7 +20,7 @@ mesonConfigurePhase() {
 
     echo "meson flags: $mesonFlags ${mesonFlagsArray[@]}"
 
-    meson build $mesonFlags "${mesonFlagsArray[@]}"
+    meson setup build $mesonFlags "${mesonFlagsArray[@]}"
```

Comment:
Just for the occasion: `build` is the name of directory or is a sub-subcommand?

As far as I remember, there was an old bug about this...

Reply:
It is the name of the directory where the generated files will be placed. Added a link to the [introducing commit](https://github.com/mesonbuild/meson/commit/3c7ab542c0c4770241eae149b0d4cd8de329aee0) for more context.

Reply:
Now you know why we deprecated `meson build`. :)

(There's no such thing as sub-subcommands. I guess it's technically possible given enough abuse of argparse, but certainly Meson doesn't use any such things.)

Reply:
I'm also a personal fan of `meson setup builddir/`, truth be told.